src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,610 @@
+
+
+#### 📌 Example(s)
+
+<CodeGroup title="" tag="POST" label="get_nft_transfers">
+```json {{ mm2MethodDecorate : true }}
+{
+  "userpass": "MM2_RPC_PASSWORD",
+  "method": "get_nft_transfers",
+  "mmrpc": "2.0",
+  "params": {
+    "chains": [
+      "POLYGON"
+    ],
+    "url": "https://moralis-proxy.komodo.earth"
+  }
+}
+```
+</CodeGroup>
+
+
+### Response Parameters
+
+| Parameter            | Type             | Description                                                                                      |
+| ---------------------| ---------------- | ------------------------------------------------------------------------------------------------ |
+| amount               | string           | Amount of tokens transferred.                                                                    |
+| block_hash           | string           | Hash of block in which transfer occurred.                                                        |
+| block_number         | integer          | Height of block in which transfer occurred.                                                      |
+| block_timestamp      | string           | Block date/time string.                                                                          |
+| chain                | string           | Chain name. One of `AVALANCHE`, `BSC`, `ETH`, `FANTOM`, or `POLYGON`.                            |
+| collection_name      | string           | May be `null`. Name of collection which token belongs to.                                        |
+| contract_type        | string           | Contract type. `ERC721` or `ERC1155`.                                                            |
+| token_address        | string           | Address of token transferred.                                                                    |
+| token_id             | string           | Token ID.                                                                                        |
+| possible_spam        | boolean          | May be `null`. If `true`, the contract address is has [been identified](https://docs.moralis.io/web3-data-api/evm/nft-spam-detection) as associated with spam, phishing attempts, or suspicious activities. |
+| transaction_hash     | string           | Transaction ID of the transfer.                                                                  |
+| transaction_index    | string           | Transaction index of the transfer.                                                               |
+| log_index            | string           | Log index of the transfer.                                                                       |
+| value                | string           | Tranfer value.                                                                                   |
+| transaction_type     | string           | Transation type. Possible values are `Single`.                                                   |
```

Comment:
In the responses I have for `get_nft_transfers`, there is `transaction_type` value `single`. I assume this is `multiple` for ERC1155 transactions involving more than one NFT?

Reply:
Yeah, I think the same. I asked moralis team about this, but thed didnt reply. 
![image](https://github.com/KomodoPlatform/komodo-docs-mdx/assets/52405288/8920233b-e46d-4086-98ae-d2cd30eecc3d)

Currently I use `safeTransferFrom` functions for both [erc721](https://eips.ethereum.org/EIPS/eip-721#specification) and [erc1155](https://eips.ethereum.org/EIPS/eip-1155#specification) standards. However, erc1155 supports `safeBatchTransferFrom` function, so  the second variant could be "Batch", but I dont know how moralis call it. well at least we use String for this field so we can catch it` pub(crate) transaction_type: String,`